src/HTMLConverter.php

```diff
@@ -111,4 +132,40 @@ public function td(string $fieldname_attribute_name): self
 
         return $clone;
     }
+
+    /**
+     * Create a DOMElement representing a single record of data
+     *
+     * @param string $rowTagName
+     * @param string $elementTagName
+     * @param array $record
+     * @param DOMDocument $doc
+     *
+     * @return DOMElement
+     */
+    private function createRecordRow(string $rowTagName, string $elementTagName, array $record, DOMDocument $doc) : DOMElement
```

Comment:
rename 
- `$rowTagName` to `$recordTagName`
- `$elementTagName ` to `$fieldTagName`

for consistency in naming with the reste of the package.

Reply:
renamed